packages/utils/src/internals/blocked.ts

```diff
@@ -5,3 +5,13 @@ export const RETRY_CSS_SELECTORS = [
     'iframe[src^="https://challenges.cloudflare.com"]',
     'div#infoDiv0 a[href*="//www.google.com/policies/terms/"]',
 ];
+
+/**
+ * Content of proxy errors that should trigger a retry, as the proxy is likely getting blocked / is malfunctioning.
+ */
+export const ROTATE_PROXY_ERRORS = [
+    'ECONNRESET',
+    'ECONNREFUSED',
+    'ERR_PROXY_CONNECTION_FAILED',
+    'Proxy responded with',
```

Comment:
this should include `ERR_TUNNEL_CONNECTION_FAILED`

https://apifier.slack.com/archives/CD0SF6KD4/p1673886555324469